addons/music/fnc_playMusic.sqf

```diff
@@ -0,0 +1,51 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_playMusic
+
+Description:
+    A function used to play music. 
+    
+Parameters:
+    1: CLASS Class to play
+    2: INT Position to play from (default: 0)
+    3: BOOL allowed to interupt already playing music (default: true)
+    
+Returns:
+    BOOL -true if started playing music
+    
+Example:
+    (begin example)
+        _bool = ["LeadTrack01_F_Bootcamp", 30, true] call CBA_fnc_playMusic;
+    (end example)
+    
+
+Author:
+    Fishy
+---------------------------------------------------------------------------- */
+#include "script_component.hpp"
+
+params [["_className","",[""]],["_time",0,[0]],["_overWrite", true,[true]]];
+
+if (_className isEqualTo "") exitWith {
+    WARNING("No class given"); 
+    false
+};
+
+If ((!_overWrite)&&{[true] call CBA_fnc_getMusicPlaying}) exitWith {false};
+
+private _return = false;
+
+private _duration = [_className,"duration"] call CBA_fnc_getMusicData;
+
+if (!isNil "_duration") then {
+    if (_time < _duration) then {
```

Comment:
you could make this an exitWith instead of a then. That way you could completly remove the _return variable and just return `false` in line 51.
But readability may suffer if you do this. not sure.